docs/api/en/math/Box3.html

```diff
@@ -12,8 +12,7 @@ <h1>[name]</h1>
 
 		<p class="desc">
 			Represents a box or cube in 3D space. The main purpose of this is to represent
-			the [link:https://en.wikipedia.org/wiki/Minimum_bounding_box Minimum Bounding Boxes]
-			for objects.
+			the world-axis-aligned bounding boxes for objects, which are used for fast collision detection. 
```

Comment:
AABBs are not only used for collision detection. I would remove this clause.

Reply:
Got it, gonna remove this statement.